Sodium/SHA256Hash.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+import Clibsodium
+
+public struct SHA256Hash {
+    public let Bytes = Int(crypto_hash_sha256_bytes())
+}
+
+extension SHA256Hash {
+    /**
+     Computes short but unpredictable (without knowing the secret key) values suitable for picking a list in a hash table for a given key.
```

Comment:
Isn't this description comment incongruent with the function body? There is no secret key here.